inc/class-upgrade.php

```diff
@@ -511,6 +511,15 @@ private function upgrade_70() {
 	 */
 	private function upgrade_71() {
 		$this->cleanup_option_data( 'wpseo_social' );
+
+		// Move the breadcrumbs setting and invert it.
+		$title_options = $this->get_option_from_database( 'wpseo_titles' );
+
+		if ( array_key_exists( 'breadcrumbs-blog-remove', $title_options ) ) {
+			WPSEO_Options::set( 'breadcrumbs-display-blog-page', ! $title_options['breadcrumbs-blog-remove'] );
```

Comment:
I think doing `WPSEO_Options::save_option( 'wpseo_titles', 'breadcrumbs-display-blog-page', ! $title_options['breadcrumbs-blog-remove']  );` might be faster.

Reply:
As this is an upgrade routine, the speed improvement (which is negligible) is not weighing up to saving the setting without having to know context about in which option it resides in.